src/elements/Segment/_Segment.js

```diff
@@ -0,0 +1,116 @@
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
+
+import META from '../../utils/Meta'
+import * as sui from '../../utils/semanticUtils'
+import {
+  useKeyOnly,
+  getUnhandledProps,
+  useKeyOrValueAndKey,
+  useValueAndKey,
+  useAlignedProp,
+} from '../../utils/propUtils'
+import Header from '../Header/Header'
+
+function _Segment(props) {
+  const {
+    _sdClass, _segmentClass,
+    className, heading, children, raised, stacked, piled, attached, aligned, floated, disabled, loading, padded,
+    basic, inverted, circular, color, clearing, compact,
+  } = props
+
+  const classes = cx(_sdClass, 'ui',
+    color,
+    useKeyOnly(raised, 'raised'),
+    useKeyOnly(stacked, 'stacked'),
+    useKeyOnly(piled, 'piled'),
+    useKeyOrValueAndKey(attached, 'attached'),
+    useAlignedProp(aligned, 'aligned'),
+    useValueAndKey(floated, 'floated'),
+    useKeyOnly(disabled, 'disabled'),
+    useKeyOnly(loading, 'loading'),
+    useKeyOrValueAndKey(padded, 'padded'),
+    useKeyOnly(basic, 'basic'),
+    useKeyOnly(inverted, 'inverted'),
+    useKeyOnly(circular, 'circular'),
+    useKeyOnly(clearing, 'clearing'),
+    useKeyOnly(compact, 'compact'),
+    className,
+    _segmentClass
+  )
+
+  const rest = getUnhandledProps(_Segment, props)
+
+  return (
+    <div className={classes} {...rest}>
+      {heading && (
+        <Header.H4>{heading}</Header.H4>
+      )}
+      {children}
+    </div>
+  )
+}
+
+_Segment._meta = {
+  library: META.library.semanticUI,
+  name: '_Segment',
+  type: META.type.element,
+  props: {
+    attached: ['top', 'bottom'],
+    floated: ['left', 'right'],
```

Comment:
I have merged #288 which includes two new sui utils: `sui.textAlignments` and `sui.floats`.